### ISSUE TITLE
[Feature] Giving us back something like the Activity Window

### ISSUE THREAD

**Issue by Variiuz** [discord/gamesdk-and-dispatch#92]
![Frame 11](https://user-images.githubusercontent.com/36776298/88704624-a23ee680-d10e-11ea-8646-d3016fd5d823.png)

When clicking the Games name, display the linked server as we see it with Announcements, either enable it for users of the Dev License, for approved games or/and verified servers; As I can imagine people buying the License to just advertise some random server it would be good for approved games to prevent abuse.

**Comment by TheGoldenCactus11** [comment]
Yeah, this would be good, along with adding the server card back to store pages, it would be very useful!

**Comment by mozartrafi** [comment]
I very agree with this one, because Discord has behaved quite strangely by making Public Server features (Verified, Partner, Game Dev) not rare anymore 😦

**Comment by advaith1** [comment]
Public/community server features were always meant to be public, some other server types just got early access to it.

**Comment by TheGoldenCactus11** [comment]
@advaith1 I think this community update sucks. Announcement channels shouldn't be available for every single server that wants one. It should be at least limited to 500 members, or more.

**Comment by mozartrafi** [comment]
> @advaith1 I think this community update sucks. Announcement channels shouldn't be available for every single server that wants one. It should be at least limited to 500 members, or more.
> 
> (Sorry for going offtopic)

I agree with this too, this update is really bad and abuseable :(

**Comment by TheCubeHamster** [comment]
> > @advaith1 I think this community update sucks. Announcement channels shouldn't be available for every single server that wants one. It should be at least limited to 500 members, or more.
> > (Sorry for going offtopic)
> 
> I agree with this too, this update is really bad and abuseable :(

Honestly I think it should be perfectly fine for the vast majority of people since it's up to members of the server to follow announcement channels, and they need manage message permissions to follow too. Shouldn't be that bad and I've been actually wanting this for a while so I can link up the announcement channels in 2 of my servers.

**Comment by mozartrafi** [comment]
Okay, it seems that it does not need to be discussed again. Let's just go back to the title of this topic.
